CHANGELOG.md

```diff
@@ -140,6 +145,7 @@ Breaking changes result in a different major. UI changes that might break custom
 
 The standard for this change log can be found [here](http://keepachangelog.com/).
 
+[0.11.1]: https://github.com/onfido/onfido-sdk-ui/compare/0.11.0...0.11.1
```

Comment:
`[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/0.11.1...master` you can add this as well